drivers/adodb-sqlite3.inc.php

```diff
@@ -596,6 +596,119 @@ function year($fld)
 		$x = "strftime('%Y',$fld)";
 		return $x;
 	}
+	
+	/**
+	* SQLite update for blob 
+	*
+	* SQLite must be a fully prepared statement, so where can either
+	* be an array (array params) or a string that we will
+	* do our best to unpack and turn ito a prepared statement
+	*
+	* @param string $table
+	* @param string $column
+	* @param string $val
+	* @param mixed  $where
+	* @param string $blobtype ignored
```

Comment:
Why add the parameter, if it's never used ?

Reply:
I'd make joke about making sure that method signatures matches the parent but I'm guessing you didn't realize that this is a core function and that the $blobtype parameter is used elsewhere.

Reply:
Duh. That's embarrassing :flushed:  That will teach me to respond quickly without looking at the big picture. ignore me...